From: Andrea Bolognani <eof@kiyuko.org>
Date: Wed, 1 Feb 2012 12:36:31 +0100
Subject: Set print job name

When printing from standard input, the job name defaults to "stdin".

This patch adds some parameters to the lp/lpr call to provide the
user with a more descriptive name.

Forwarded: yes
---
 rhinote.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rhinote.py b/rhinote.py
index b49a88e..ac1f2e7 100644
--- a/rhinote.py
+++ b/rhinote.py
@@ -172,9 +172,10 @@ http://rhinote.tuxfamily.org
         ]
         # Find print and format commands
         self.printcommand = self.which('lp')
+        self.printargs = ['-t', 'Rhinote file']
         if not self.printcommand:
             self.printcommand = self.which('lpr')
-        self.printargs = []
+            self.printargs = ['-T', 'Rhinote file']
         self.formatcommand = self.which('enscript')
         self.formatargs = ['-B', '--word-wrap', '-o', '-']
 
